plugins/obs-qsv11/obs-qsv11-plugin-main.c

```diff
@@ -60,11 +60,15 @@ OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("obs-qsv11", "en-US")
 MODULE_EXPORT const char *obs_module_description(void)
 {
-	return "Intel Quick Sync Video H.264 encoder support for Windows";
+	return "Intel Quick Sync Video encoder support for Windows";
 }
 
 extern struct obs_encoder_info obs_qsv_encoder;
 extern struct obs_encoder_info obs_qsv_encoder_tex;
+#ifdef ENABLE_HEVC
+extern struct obs_encoder_info obs_qsv_hevc_encoder;
```

Comment:
We suggest only enabling HEVC for iGPUs from 10th Gen platforms (ICL+) and Arc graphics. A combination of CPU ID check for iGPU cases on 10th Gen+ and MFXVideoCORE_QueryPlatform to check mfxPlatform->MediaAdapterType can help detect Arc graphics.

Reply:
How about something like this?
```
bool allow_hevc =
    (media == MFX_MEDIA_DISCRETE) ||
    ((media == MFX_MEDIA_INTEGRATED) && (platform >= MFX_PLATFORM_ICELAKE);
```

Reply:
I updated the code to check inside qsv_encoder_open. Resolve the conversation if you agree.

Reply:
Did a few quick tests on my side - this looks good. There is no option for me to resolve the conversation though, only comment. Can you close it please?